profiles/common/modules/custom/tmgmt_poetry/tmgmt_poetry_mock/response_templates/poetry-send-status.tpl.php

```diff
@@ -0,0 +1,46 @@
+<?php
+/**
+ * @file
+ * Template file for the POETRY send status message.
+ *
+ * Available custom variables:
+ * - $demande_id: An array with requester data.
+ * - $languages: an array with language codes.
+ * - $format: string with attribution format.
+ * - $status_code: job status.
+ * - $request_status_msg: request status message.
+ * - $demande_status_msg: 'demande' status message.
+ * - $lg_code: language code.
+ */
+?>
+<?php print '<?xml version="1.0" encoding="UTF-8"?>' . PHP_EOL; ?>
+<POETRY xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="">
+  <request communication="synchrone" id="<?php print rand(1000000, 9999999); ?>" type="status">
+    <demandeId>
+      <codeDemandeur><?php print $demande_id['codeDemandeur']; ?></codeDemandeur>
+      <annee><?php print $demande_id['annee']; ?></annee>
+      <numero><?php print $demande_id['numero']; ?></numero>
+      <version><?php print $demande_id['version']; ?></version>
+      <partie><?php print $demande_id['partie']; ?></partie>
+      <produit><?php print $demande_id['produit']; ?></produit>
+    </demandeId>
+    <status code="0" type="request">
+      <statusDate><?php print date('d/m/Y'); ?></statusDate>
```

Comment:
No. 6 https://webgate.ec.europa.eu/fpfis/wikis/display/MULTISITE/Guide+into+code+reviewing+for+Drupal+7#GuideintocodereviewingforDrupal7-known_issuesQAknownissues -> all of the usage of 'date' function (i don't repeat it everywhere from now on)